roles/libvirt_manager/tasks/ocp_layout.yml

```diff
@@ -73,13 +74,30 @@
 
     - name: Create blank images for OCP cluster resources
       vars:
+        _matcher: >-
+          {{
+            '(cifmw-)?' ~
+            '([a-z0-9]+)-[0-9]+'
+          }}
+        _ocp_master_matcher: >-
+          {{
+            '^ocp-(\d+)$'
+          }}
         vm: "{{ _vm.key }}"
         vm_type: "{{ _vm.value }}"
         vm_data: >-
           {{
             _ocp_layout.vms[vm_type]
           }}
-        is_base_img: true
+        # If $vm is an OCP master node, capture it's index
+        # and craft the _vm_img varible, examples:
+        # ocp-0 -> ocp_master-0.qcow (match)
+        # ocp_worker-0 -> ocp_worker-0.qcow (pass)
+        # compute-1 -> compute-1.qcow (pass)
+        # _vm_img: "{{ vm | regex_search(_ocp_master_matcher) | ternary(vm | regex_replace(_ocp_master_matcher, 'ocp_master-\\1.qcow2'), vm ~ '.qcow2') }}"
```

Comment:
image name should be converged to use hyphens, not dashes.

Reply:
Removed